lib/alias.ps1

```diff
@@ -0,0 +1,83 @@
+. "$PSScriptRoot\..\lib\install.ps1" # shim related
+
+$script:config_alias = 'alias'
```

Comment:
Why create a script var?

Reply:
Uh oh, this was moved from the original code and left unchanged. Will change it.